### PR TITLE
Extended GA flow checking more commands.

### PR DESCRIPTION
**Pull request by andreas-kupries**
- Commands checked: `info`, `create-org`, `orgs`, `push`.
 - Inactive preps for `apps`, `delete`.

### CODE REVIEW

**Comment by andreas-kupries** [comment]
Yes, the sleep looks to be necessary. Not sure how small we can make it. The value set right now is based on my local experience, where I knew it had worked.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -55,15 +55,35 @@ jobs:
...
+          ./cli/dist/carrier-linux-amd64 orgs
+      - name: Create Organization
+        run: |
+          ./cli/dist/carrier-linux-amd64 create-org myself
+      - name: Carrier Orgs
+        run: |
+          ./cli/dist/carrier-linux-amd64 orgs
+      - name: Push an App
+        run: |
+          sleep 8m ;# Wait for tekton to be usable
```
Comment (jimmykarily) [inline .github/workflows/main.yml:72]: Shouldn't we better add waiting loops in the install command to make sure Tekton is usable? We can do it on another PR if you prefer.

Comment (andreas-kupries) [reply .github/workflows/main.yml:72]: I agree that having some kind of waiting in the installer would be best. I have however no idea wait exactly we have to wait for in tekton, so for this one I simply did the hack to get around whatever this is. A new PR changing tekton (and removing the sleep here) would indeed be my preference.

**Review by jimmykarily — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (jimmykarily) [reply .github/workflows/main.yml:72]: ok let's do it like that

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -55,15 +55,35 @@ jobs:
...
+          ./cli/dist/carrier-linux-amd64 create-org workspace ;# FUTURE: Part of install.
+          git config --global user.email "ci@suse.dev"
```
Comment (jimmykarily) [inline .github/workflows/main.yml:74]: I guess this is a fix for this: https://github.com/SUSE/carrier/issues/12
I'm currently fixing that issue. If this is merged first I will remove these 2 lines so that we check if my fix works.
If this is merged last, we should remove these lines on this PR.

Comment (jimmykarily) [reply .github/workflows/main.yml:74]: https://github.com/SUSE/carrier/pull/58

Comment (andreas-kupries) [reply .github/workflows/main.yml:74]: This is now merged.

Comment (jimmykarily) [reply .github/workflows/main.yml:74]: So I can delete these lines I guess.

Comment (jimmykarily) [reply .github/workflows/main.yml:74]: https://github.com/SUSE/carrier/pull/60
